packages/components/src/components/block-section/block-section.tsx

```diff
@@ -97,7 +97,7 @@ export class BlockSection extends LitElement {
   /**
    * Displays a status-related indicator icon.
    *
-   * @deprecated Use `icon-start` instead.
+   * @deprecated in v2.8.1, removal target v6.0 - Use `icon-start` instead.
```

Comment:
Same as above consistency:
```suggestion
   * @deprecated in v2.8.1, removal target v6.0.0 - Use `icon-start` instead.
```